include/gazebo_opticalFlow_plugin.h

```diff
@@ -34,6 +34,11 @@
 #include <iostream>
 #include <boost/timer/timer.hpp>
 
+extern "C" {
```

Comment:
@jgoppert there seems to have been a misunderstanding. what I meant was to move `extern "C"` into the `flow_settings.h` and `flow.h` headers around all function declarations. That's the common way to do it, as the user of the header should not need to bother if it's C or C++.